tests/unit/style/properties/serialization.rs

```diff
@@ -1231,4 +1231,26 @@ mod shorthand_serialization {
             assert_eq!(serialization, block_text);
         }
     }
+      mod effects {
```

Comment:
I think something went wrong and it created merge master branch. I don't know how to undo that but last commit to be considered is in box-shadow is now serialized properly. Let me know if I need to do anything to fix it. Sorry if this messes something!!